source/AI.cpp

```diff
@@ -3655,7 +3655,7 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player, Command &activeCommand
 	}
 	else if(activeCommands.Has(Command::JUMP | Command::FLEET_JUMP))
 	{
-		if(!ship.GetTargetSystem() && !isWormhole)
+		if((!ship.GetTargetSystem() || player.TravelPlan().empty()) && !isWormhole)
```

Comment:
this line could need a comment now that it is more complex

Reply:
You could chop it down to if(player.TravelPlan().empty() && !isWormhole). GetTargetSystem is no longer required

Reply:
Line has been trimmed as described. Thanks for the suggestion!